### PR TITLE
added s3 loading

### PR DESCRIPTION
**Pull request by Sefadzi**

### CODE REVIEW

File: scripts/ento/ento_storage/R/app_server.R [hunk 1]
```diff
@@ -900,7 +900,7 @@ app_server <- function( input, output, session ) {
   
   #send to storage
   observeEvent(input$c_store, {
-    
+    browser()
```
Comment (katekimani) [inline scripts/ento/ento_storage/R/app_server.R:903]: Debug feature needs clean up

File: scripts/ento/ento_storage/R/app_server.R [hunk 2]
```diff
@@ -1060,62 +1060,62 @@ app_server <- function( input, output, session ) {
...
-  
-  output$leq <- DT::renderDT({
-    isolate(le_rv$df)
-    DT::datatable(date_filtered_le())
-  })
+  # ##-----Household Questionnaire
+  # hh_rv <- reactiveValues(df = data.frame())
+  # 
+  # observeEvent(shiny::reactiveTimer(4000000)(), {
+  #   hh_rv$df = isolate({
```
Comment (katekimani) [inline scripts/ento/ento_storage/R/app_server.R:1067]: If the dashboard report has been moved ok, please clean up this commented out code.

File: scripts/ento/ento_storage/R/fct_A3storage.R [hunk 3]
```diff
@@ -2,37 +2,40 @@
...
   
+
   
   if(nrow(entoa3_df)>=1){
+    
     storage <- entoa3_df %>%
       dplyr::select_at(dplyr::vars(contains("how_many_stored")))
     
+    storage <- entoa3_df %>%
+      dplyr::select_at(dplyr::vars(contains("how_many_stored")))%>%
```
Comment (katekimani) [inline scripts/ento/ento_storage/R/fct_A3storage.R:36]: Delete the L35 and L36 this is duplicating

File: scripts/ento/ento_storage/R/fct_C3storage.R [hunk 4]
```diff
@@ -186,11 +143,13 @@ C3storage = function(){
...
                        "Sample boxes ID and link to QR code",
                        "Position in the Box")
+
+    
   }else{
     
     rep_df <- data.frame(
       "Date" = as.Date(character()),
-      "HH ID" = character(),
+      "Cluster" = character(),
```
Comment (katekimani) [inline scripts/ento/ento_storage/R/fct_C3storage.R:152]: Why the change to cluster here? I believe the user review done approved the page with `HH ID`

File: scripts/ento/ento_storage/R/fct_C4_preprocess.R [hunk 5]
```diff
@@ -70,7 +102,7 @@ C4_preprocess = function(c4output, lp) {
     
     names(a5) = c("Date",
                   "Cluster",
-                  "Collection location (OR)",
+                  "Collection location",
```
Comment (katekimani) [inline scripts/ento/ento_storage/R/fct_C4_preprocess.R:105]: 👍

File: scripts/ento/ento_storage/R/fct_getEntoDataC4.R [hunk 6]
```diff
@@ -99,7 +73,7 @@ getEntoDataC4 <- function(){
                           "D7_ovi",
                           "D7_death",
                           "Species Complex",
-                          "Sample tude ID and link to QR Code",
+                          "Sample tube ID and link to QR code",
```
Comment (katekimani) [inline scripts/ento/ento_storage/R/fct_getEntoDataC4.R:76]: 👍

**Review by katekimani — CHANGES_REQUESTED** [review]
Please see comments inline. Also, would you deploy the new changes to https://sefadzi.shinyapps.io/entodashboard/ for me to test before we deploy prod.

Also, the readme on how to deploy, is this updated?

File: scripts/ento/ento_storage/R/app_server.R (hunk 1 above)
Comment (Sefadzi) [reply scripts/ento/ento_storage/R/app_server.R:903]: alright

**Review by katekimani — CHANGES_REQUESTED** [review]
@Sefadzi  What is the reason for the `forms/*` in the PR? I am unclear as to how they ended up with changes on your branch, maybe update your branch with latest master changes?

**Pull request merged**